langchain/src/agents/tools/vectorstore.ts

```diff
@@ -0,0 +1,39 @@
+import { VectorStore } from "../../vectorstores/index.js";
+import { BaseLLM } from "../../llms/index.js";
+import { VectorDBQAChain } from "../../chains/index.js";
+import { Tool } from "./base.js";
+
+interface VectorStoreTool {
+  vectorStore: VectorStore;
+  llm: BaseLLM;
+}
+
+export class VectorStoreQATool extends Tool implements VectorStoreTool {
+  vectorStore: VectorStore;
+
+  llm: BaseLLM;
+
+  name: string;
+
+  description: string;
+
+  constructor(name: string, description: string, fields: VectorStoreTool) {
+    super();
+    this.name = name;
+    this.description = description;
+    this.vectorStore = fields.vectorStore;
+    this.llm = fields.llm;
+  }
+
+  static getDescription(name: string, description: string): string {
+    return `Useful for when you need to answer questions about ${name}. "
```

Comment:
I think the `"` shouldn't be here? In a ` string double quotes will just show up as any other character (so will the whitespace at the beginning of each line).

Reply:
yup good catch